installer/util/aws.go

```diff
@@ -61,6 +62,7 @@ func EnsureBucketExists(bucketName string, s3Client *s3.Client) error {
 		if err != nil {
 			return fmt.Errorf("could not create S3 bucket %s: %w", bucketName, err)
 		}
+		time.Sleep(5 * time.Second) // give AWS time to create bucket
```

Comment:
The sdk provides a waiter for this one 

See: https://github.com/aws/aws-sdk-go/blob/v1.36.29/service/s3/waiters.go#L16

Reply:
We use sdkv2 - could not find a way to do a waiter there.

Reply:
Apparently version of aws sdk v2 v0.31 has it. Migrated to v1.0.0 - possibly no more api breakage!